content/20.treatments.md

```diff
@@ -401,18 +403,22 @@ In influenza, SPM promote antiviral B lymphocytic activities [@doi:10.4049/jimmu
 It is hypothesized that SPM may aid in the resolution of the cytokine storm and pulmonary inflammation associated with COVID-19 [@doi:10.1007/s10555-020-09889-4].
 However, not all studies are in agreement that n-3 PUFA are effective against infections [@doi:10.3945/jn.109.108027].
 The effectiveness of n-3 PUFA against infections would be dependent on the dosage, timing, and the specific pathogens responsible [@doi:10.1016/j.jinf.2016.10.001].
+On another level, there is still the question of whether fish oils can raise the levels of SPM levels upon ingestion and in response to acute inflammation in humans [@doi: 10.1194/jlr.M060392].
 
-One pathological aspect of COVID-19 infection that was late to be recognized is the increased risk of thrombotic complications [@doi:10.1016/j.thromres.2020.04.013]. 
+The increased risk of thrombotic complications in COVID-19 infected patients was relatively late to be reported in comparison to other COVID-19 manifestations [@doi:10.1016/j.thromres.2020.04.013; @doi:10.1182/blood.2020006000].
 Considering that there is significant evidence that n-3 fatty acids and other fish oil-derived lipids possess antithrombotic properties and anti-inflammatory properties [@doi:10.1016/j.prostaglandins.2018.09.005; @doi:10.1016/j.blre.2020.100694], they may have therapeutic value against the prothrombotic complications of COVID-19. 
-This may be particularly important because patients receiving pharmaceutical antiplatelet therapies may become infected.
-Consequently, there is concern among the medical community that the use of investigational therapeutics on COVID-19 patients already on antiplatelet therapies due to a pre-existing comorbidities may lead to issues with dosing and drug choice, and/or negative drug-drug interactions [@doi:10.1016/j.jacc.2020.04.031].
+This may be particularly important for patients already receiving pharmaceutical antiplatelet therapies who may become infected.
+This is because there is concern among the medical community that the use of investigational therapeutics on COVID-19 patients already on antiplatelet therapies due to a pre-existing comorbidities may lead to issues with dosing and drug choice, and/or negative drug-drug interactions [@doi:10.1016/j.jacc.2020.04.031].
 As a result, the use of other therapeutics such as dietary sources of n-3 fatty acids or nutraceuticals with antiplatelet activities may be beneficial and warrant further investigation.
+A recently developed highly purified derivative of EPA known as icosapent ethyl (VascepaTM) [@doi:10.1056/NEJMoa1812792] is currently recruiting participants for randomized controlled clinical trial to investigate its anti-inflammatory activity in COVID-19 positive patients [@clinicaltrials:NCT04412018]. 
```

Comment:
```suggestion
A new clinical trial [@clinicaltrials:NCT04412018] is currently recruiting COVID-19 positive patients to investigate the anti-inflammatory activity of a recently developed, highly purified derivative of EPA known as icosapent ethyl (Vascepa ^TM^) [@doi:10.1056/NEJMoa1812792].  
```
Hopefully this is formatted correctly, I haven't done superscripts in Manubot before! I based it on [this](https://github.com/manubot/rootstock/blob/master/content/02.delete-me.md).